ArchiSteamFarm/Bot.cs

```diff
@@ -1518,7 +1518,7 @@ public sealed class Bot : IAsyncDisposable {
 					ASF.ArchiLogger.LogGenericWarning(string.Format(Strings.ErrorIsInvalid, nameof(name)));
 				}
 
-				if (invalid && !string.IsNullOrEmpty(key)) {
+				if (invalid && !(key is null)) {
```

Comment:
What is wrong with `&& (key != null)` ?

Reply:
You can also remove null-forgiving operator `!` from key below, as roslyn will be smart enough to realize that it's checked at this point.

Reply:
Nothing is wrong with `!=` but I usually prefer `is`-keyword and in current Roslyn versions they both result in the [same IL Code](https://sharplab.io/#v2:C4LglgNgNAJiDUAfAAgJgIwFgBQyDMABGgQMIEDeOB1BARgPb0QECy6AFPbQFYCmAxsAIAPAJQEAvAD4RBMAGcCAOwCuECAG4cASAZNWqTjwFCxkmcMkTlazTgC+QA==) when there is no overloaded equality operator (which there is not).

Yes removing that operator is something I overlooked

Reply:
Use `!=` for consistency then, `!(something is type)` is less readable and doesn't follow the rest of the project.